EngineLayer/FdrAnalysis/PEPValueAnalysisGeneric.cs

```diff
@@ -71,12 +91,8 @@ public static string ComputePEPValuesForAllPSMsGeneric(List<PeptideSpectralMatch
                     //Here we compute the pepvalue predection for each ambiguous peptide in a PSM. Ambiguous peptides with lower pepvalue predictions are removed from the PSM.
```

Comment:
It seems like we have another ambiguous peptide removing mechanism in Protein parsimony. Although they are at different levels. Could we include Protein information in the model? It seems complicated for now.